test/test_functions

```diff
@@ -1181,3 +1188,125 @@ cp_usrbin() {
     sudo chmod 0444 $f || return 3
   done
 }
+
+
+# waits until cvmfs is mounted
+wait_fqrn_mount() {
+  local waits=0
+  while ! mount | grep "/cvmfs/${FQRN}" > /dev/null 2>&1; do
+    waits=$(( waits + 1 ))
+    if [ "$waits" -ge 10 ]; then
+      die "${FQRN} could not be mounted (not superuser?)"
+    fi
+    sleep 1
+  done
+}
+
+
+# runs the command once in order to get a warm cache
+create_warm_cache() {
+  local workspace=$1
+  cvmfs2 -f -o config=cvmfs.conf,disable_watchdog,simple_options_parsing $FQRN /cvmfs/$FQRN &> /dev/null &
+  wait_fqrn_mount
+
+  bash -c "cvmfs_run_benchmark" > /dev/null 2>&1
+  umount /cvmfs/$FQRN
+  return 0
+}
+
+
+# creates the default cvmfs.conf file for normal testing
+#
+# @param destination_file  absolute path of the directory
+create_default_cvmfs_config() {
+  local destination_directory="$1"
+  echo 'CVMFS_CACHE_BASE=/root/cvmfs_testing/cache
```

Comment:
That would be fine with me, as long as the benchmarks can be executed with a ramdisk cache from Jenkinks in one way or another.  I think it would be an important option for the benchmarks because it cancels out the uncertainties from accessing the hard disk.